sdk/core/azure-core-tracing-opentelemetry/src/opentelemetry.cpp

```diff
@@ -0,0 +1,189 @@
+
```

Comment:
Missing license comment on top of file.

Reply:
W$W#$#..

This should be covered in the CI pipeline.

Reply:
This is now: https://github.com/Azure/azure-sdk-for-cpp/issues/3675